File: app/src/main/java/org/mifos/mobilebanking/ui/views/HomeView.java
package org.mifos.mobilebanking.ui.views;

import android.graphics.Bitmap;

import org.mifos.mobilebanking.ui.views.base.MVPView;

/**
 * Created by dilpreet on 19/6/17.
 */

public interface HomeView extends MVPView {

    void showUserDetails(String userName);

    void showUserImageTextDrawable();

    void showUserImage(Bitmap bitmap);

    void showNotificationCount(int count);

    void showError(String errorMessage);

    void showUserImageNotFound();

}

File: app/src/main/java/org/mifos/mobile/ui/views/HomeView.kt
package org.mifos.mobile.ui.views

import android.graphics.Bitmap
import org.mifos.mobile.ui.views.base.MVPView

/**
 * Created by dilpreet on 19/6/17.
 */
interface HomeView : MVPView {
    fun showUserDetails(userName: String?)
    fun showUserImageTextDrawable()
    fun showUserImage(bitmap: Bitmap?)
    fun showNotificationCount(count: Int)
    fun showError(errorMessage: String?)
    fun showUserImageNotFound()
}